install-atomicsfolder.ps1

```diff
@@ -86,7 +86,8 @@ function Install-AtomicsFolder {
                 $ms = New-Object IO.MemoryStream
                 [Net.ServicePointManager]::SecurityProtocol = ([Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls11 -bor [Net.SecurityProtocolType]::Tls12)
 
-                $webClient = New-Object ExtendedWebClient; 
+                $webClient = New-Object HttpClient;
+		$webClient.Timeout = TimeSpan.FromMinutes(3);
                 $webClient.OpenRead($url).copyto($ms)
```

Comment:
[`HttpClient`](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=net-7.0) doesn't have OpenRead method as well.

Reply:
I just ran this on Windows under Powershell 7.3.4 and it works

`pwsh -Command "IEX (IWR 'https://raw.githubusercontent.com/mbaran5/invoke-atomicredteam/master/install-atomicredteam.ps1' -UseBasicParsing); Install-AtomicRedTeam -RepoOwner mbaran5 -getAtomics -Force -Verbose"   `

```
VERBOSE: Importing function 'Stop-AtomicGUI'.
VERBOSE: Installing Atomics Folder
VERBOSE: Directory Creation
VERBOSE: Beginning download of atomics folder from Github
VERBOSE: HTTP/1.1 GET with 0-byte payload
VERBOSE: received -byte response of content type application/zip
VERBOSE: Extracting ART to C:\AtomicRedTeam
Installation of Invoke-AtomicRedTeam is complete. You can now use the Invoke-AtomicTest function
See Wiki at https://github.com/mbaran5/invoke-atomicredteam/wiki for complete details
PS C:\Users\matt>
```

Reply:
![image](https://github.com/redcanaryco/invoke-atomicredteam/assets/583865/5d35f8de-bd3e-4133-9ec8-010ac1748804)

Reply:
Forgot to mention my environment:

Tried it with MacOS powershell core(pwsh) version  7.3.4

Reply:
let me fire up my Mac. Perhaps the MacOS pwsh doesn't support that function while Windows and Linux do.

Reply:
Sorry. never mind. It works. Tried again after clearing my powershell profile. Will test it again on Ubuntu and Windows and will approve this PR. Thanks for the contribution.

Reply:
awesome! Thanks!

Reply:
Found the issue. `HttpClient` is used only when -NoPayloads option is enabled. Can you try the following command ? 

```
pwsh -Command "IEX (IWR 'https://raw.githubusercontent.com/mbaran5/invoke-atomicredteam/master/install-atomicredteam.ps1' -UseBasicParsing); Install-AtomicRedTeam -RepoOwner mbaran5 -getAtomics -Force -Verbose -NoPayloads"   
```

Reply:
Ah, same here. It must've been pulling out the old custom function that allowed it run without the -NoPayloads flag. I'll close the PR and maybe try to spend a bit of time on it later. I hadn't tried it with the -NoPayloads.